dao/mongo.go

```diff
@@ -214,119 +213,85 @@ func (m *MongoService) UpdateInsolvencyPractitioners(practitionersResource model
 
 	return http.StatusNoContent, nil
 }
+ 
+// GetInsolvencyPractitionerByTransactionID gets a list of all practitioners for an insolvency case with the specified transactionID
+func (m *MongoService) GetInsolvencyPractitionerByTransactionID(transactionID string) (map[string]string, string, error) {
```

Comment:
The return types here don't seem to fit with the dao package - all other get operations return just one or more Dao/Dto objects, in line with the package's purpose as an interface for the persistence layer.  Are we doing conversions here that would be better handled by the calling functions?

Reply:
Good point resolved.Thanks